Functions/xASL_bids_BIDS2Legacy.m

```diff
@@ -284,13 +284,14 @@
 if ~isempty(ListASL4D)
     for iList=1:numel(ListASL4D)
         xASL_bids_parseM0(ListASL4D{iList});
+        [~, currentNifti] = fileparts(ListASL4D{iList});
+        fprintf('M0 parsed for %s %s ...\n', studyName, currentNifti);
     end
-    fprintf('%s\n', ['M0 parsed for ' ListASL4D{iList}]);
 else
     warning(['No ASL4D file found in ' pathLegacy]);
 end
 
-%% 10. Create DataPar.json
+%% 10. Create dataPar.json
 
 % Write DataParFile if it does not exist already
 if ~(xASL_exist(fullfile(pathLegacy, 'dataPar.json'),'file')==2)
```

Comment:
You can change this into:

`if ~xASL_exist(fullfile(pathLegacy, 'dataPar.json'), 'file')`

so in Matlab you can omit the outer () brackets, and with 'file' you don't need to ask if it's equal to 2.

Reply:
0d367c3 👍